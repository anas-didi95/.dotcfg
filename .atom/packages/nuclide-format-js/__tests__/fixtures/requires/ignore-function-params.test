function one(two, {three}) {
  two();
  three();
  four(five);
}

var six = (seven, eight) => ({seven, eight, nine});
