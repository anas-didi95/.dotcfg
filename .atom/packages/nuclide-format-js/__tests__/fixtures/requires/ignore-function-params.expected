const five = require('five');
const four = require('four');
const nine = require('nine');

function one(two, {three}) {
  two();
  three();
  four(five);
}

var six = (seven, eight) => ({seven, eight, nine});
